pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/visitor/JavaCodeWriter.java

```diff
@@ -44,11 +44,28 @@
 
 public class JavaCodeWriter extends AstVisitor<JavaCodeWriter.CodeToResolve> {
 
+    private Set<String> javaTypesUsed = new HashSet<>();
+
+    private void addToImports(String fullJavaType) {}
```

Comment:
It's just dummy method? It looks to remove the method if it would be used any more.

Reply:
It is unused code and should be removed.
Thank you.